Spine/DeserializeOperation.swift

```diff
@@ -203,7 +203,9 @@ class DeserializeOperation: NSOperation {
 			if let extractedValue: AnyObject = self.extractAttribute(serializedData, key: key) {
 				let formattedValue: AnyObject = self.valueFormatters.unformat(extractedValue, forAttribute: field)
 				resource.setValue(formattedValue, forField: field.name)
-			}
+            } else {
```

Comment:
What happens when an attribute is not included in the JSON? Is a `nil` value set on the Spine resource or is it left as is?